docs/source/working_with_aiida/groups.rst

```diff
@@ -29,11 +30,18 @@ be performed with Groups:
 
 
 2. **List available Groups.**
-      > verdi group list
 
-    .. note:: By default ``verdi group list`` only shows groups of type *user*.
-      In case you want to show groups of other type use ``-t/--type`` options. If
-      you want to show groups of all types, use ``-a/--all-types`` option.
+    Example::
+
+      verdi group list
+
+    By default ``verdi group list`` only shows groups of type *user*.
+    In case you want to show groups of other type use ``-t/--type`` option. If
+    you want to show groups of all types, use ``-a/--all-types`` option.
```

Comment:
use ``-a/--all-types`` option -> use the ``-a/--all-types`` option

Reply:
done